Makefile

```diff
@@ -0,0 +1,100 @@
+# Defaults
+REV:=$(shell git rev-parse --short HEAD)
+DATE:=$(shell date +%Y.%m.%d-%H.%M.%S)
+BRANCH:=$(shell git rev-parse --abbrev-ref HEAD)
+COMMIT:=$(MODULE)_$(INFRA_VALUES)_$(DATE)_$(REV)
+
+CURRENT_DIR_PATH:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+
+# Inputs
+REGION ?= ${REGION}
+PROJECT_ID ?= ${PROJECT_ID}
+DEPLOYMENT_ID ?= ${DEPLOYMENT_ID}
+ZONAL ?= ${ZONAL}
+
+TF_STATE_BUCKET ?= ${TF_STATE_BUCKET}
+
+# Terraform Setup
+TF_MODULE_PATH:=test
+TF_PLAN_FILE:=test_$(DEPLOYMENT_ID)_$(REV)
+
+.PHONY: clean
+
+pre-commit: clean
```

Comment:
It would be helpful to put the help target and help text per-target like we do in other repos: https://github.com/astronomer/ci-images/blob/886a106d/Makefile#L22-L24